tests/e2e/Pages/MapComponent.ts

```diff
@@ -115,6 +120,36 @@ class MapComponent extends DashboardPage {
       await expect(adminBoundaries.nth(i)).toBeVisible();
     }
   }
+
+  async turnOffLayer({ layerName }: { layerName: string }) {
+    // Remove Glofas station from the map (in case the mock is for floods)
+    await this.layerMenuToggle.click();
+    const getLayerRow = this.page
+      .getByTestId('matrix-layer-name')
+      .filter({ hasText: layerName });
+    const layerCheckbox = getLayerRow.locator(this.layerCheckbox);
+    await layerCheckbox.click();
+  }
+
+  async assertAggregateTitleOnHoverOverMap() {
+    // Declare component
+    const aggregates = new AggregatesComponent(this.page);
+
+    // Wait for the page to load
+    await this.page.waitForLoadState('networkidle');
+    await this.page.waitForLoadState('domcontentloaded');
+    await this.page.waitForSelector('.leaflet-interactive');
+
+    // Assert the that Aggregates title is visible and does not contain the text 'National View'
+    // 20 is the more or less number of admin boundaries that can be hovered over with Playwright to succesfully assert the title
```

Comment:
I don't understand this magic number 20. Is it not sufficient to test one admin boundary and assume the others work the same way?

I feel I'm missing some testing-specific science here so correct me if I'm wrong.

Reply:
I thought about it and it does not make sense to put any number there. Initially I wanted it to loop over all the components but I was facing a huge timeouts. I'll change it to validate to one. Also not all disaster types will have 20 regions displayed.

Reply:
Thank you for clarifying @Piotrk39